### PR TITLE
Adds support for pretty printed equality comparison 

### PR DESCRIPTION
**Pull request by Baccata**
Leveraging Expecty's awesomeness one more time. 

```scala
expect.same("foo foo foo", "foo bar foo")
```

Screenshot 2021-01-10 at 16 28 49

Just need to wait for [this](https://github.com/eed3si9n/expecty/pull/41) to be released

### CODE REVIEW

**Comment by keynmol** [comment]
hate to say it, but yo do you want to add a dogfood test for how this is rendered?

**Comment by keynmol** [comment]
And honestly, anything stopping us from using this in dog food tests? instead of my bad `expectEquals` (although I don't know how this works with whitespace)

**Comment by Baccata** [comment]
> hate to say it, but yo do you want to add a dogfood test for how this is rendered?

will do over the weekend, or next week 

> And honestly, anything stopping us from using this in dog food tests 

my own laziness ?

File: modules/core/src/weaver/Expectations.scala [hunk 1]
```diff
@@ -134,15 +134,6 @@ object Expectations {
     def inEach[L[_], A](la: L[A])(f: A => Expectations)(
         implicit L: Foldable[L]): Expectations = forEach(la)(f)
 
-    def verify(condition: Boolean, hint: String)(
-        implicit pos: SourceLocation): Expectations =
-      if (condition) success
```
Comment (keynmol) [inline modules/core/src/weaver/Expectations.scala:139]: sure you want to remove this?

Comment (Baccata) [reply modules/core/src/weaver/Expectations.scala:139]: oops, probably not :s

**Review by keynmol — APPROVED** [review]

File: modules/core/src/weaver/internals/ExpectyListener.scala [hunk 2]
```diff
@@ -1,11 +1,13 @@
 package weaver
+package internals
 
 import cats.data.{ NonEmptyList, ValidatedNel }
 import cats.syntax.all._
 
 import com.eed3si9n.expecty._
 
-class ExpectyListener extends RecorderListener[Boolean, Expectations] {
+private[weaver] class ExpectyListener
```
Comment (keynmol) [inline modules/core/src/weaver/internals/ExpectyListener.scala:9]: I remember you asked me on the scala 3 PR if making it package private will work outside of weaver package (and it does, but we're not sure why)

Comment (Baccata) [reply modules/core/src/weaver/internals/ExpectyListener.scala:9]: Are you not confusing the `ExpectyListener` class with the `Expecty` class ?

Comment (keynmol) [reply modules/core/src/weaver/internals/ExpectyListener.scala:9]: image

**Pull request merged**